### PR TITLE
Quo animated list items

### PR DESCRIPTION
**Pull request by Ferossgp**

### CODE REVIEW

File: src/quo/components/animated/pressable.cljs [hunk 1]
```diff
@@ -97,3 +100,5 @@
                                           :border-radius    border-radius})}]
            (into [animated/view {:style foreground}]
                  children)]]]))))
+
+
```
Comment (flexsurfer) [inline src/quo/components/animated/pressable.cljs]: extra line

File: src/quo/components/list/item.cljs [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                                    :resize-mode     :center
+                                    :color           (:icon-02 @colors/theme)}]])])
+
+(defn list-item
+  [{:keys [theme accessory disabled subtitle-max-lines icon title
+           subtitle active on-press on-long-press chevron size
+           accessory-text]
+    :or   {subtitle-max-lines 1
+           theme              :main}}]
+  (let [theme' (if disabled :disabled theme)
```
Comment (flexsurfer) [inline src/quo/components/list/item.cljs]: yeah it might be hard, but better to give names and do not use '

File: src/quo/previews/lists.cljs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  (:require [reagent.core :as reagent]
+            [quo.core :as quo]
+            [quo.react-native :as rn]
+            [quo.design-system.colors :as colors]
+            [quo.previews.preview :as preview]))
+
+(def all-props (preview/list-comp [] {}))
+
+(defn avatar []
+  [rn/view {:border-radius    20
```
Comment (flexsurfer) [inline src/quo/previews/lists.cljs:11]: probably  it would be better to avoid confusing move all components to quo, even view

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
@Ferossgp is that builds to tests something regarding fonts/view? Because this is ho it looks after installation:

Screenshot 2020-05-25 at 16 03 04

**Comment by Ferossgp** [comment]
@Serhy yes, this is Quo app, will restore Status app after design-review

**Comment by errorists** [comment]
@Ferossgp when design review?

**Comment by Ferossgp** [comment]
@Serhy can you please confirm that failed tests are not related to PR?

**Comment by Serhy** [comment]
Yes, they are not related to this PR, @Ferossgp

**Pull request merged**